java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -60,7 +60,7 @@ import kotlinx.coroutines.runBlocking
  * the [IStorageService] interface when bound-to by a client.
  */
 open class StorageService : ResurrectorService() {
-    private val coroutineContext = Dispatchers.IO + CoroutineName("StorageService")
+    private val coroutineContext = Dispatchers.Default + CoroutineName("StorageService")
```

Comment:
Don't we want to  supply this via a protected open fun or val that can be overriden by a subclass?

Reply:
Yeah but I prefer do it in a different PR.